trino/trino.go

```diff
@@ -227,6 +230,7 @@ func (c *Config) FormatDSN() (string, error) {
 		"session_properties": strings.Join(sessionkv, ","),
 		"extra_credentials":  strings.Join(credkv, ","),
 		"custom_client":      c.CustomClientName,
+		"bearer_auth":       c.BearerAuth,
```

Comment:
This needs a test, ideally an integration one, where you would configure the Trino server to accept this kind of authentication.

Reply:
I would like this feature , if its not too much to ask, can I continue on this PR?

Reply:
Sure, you can copy this branch into your fork and open up a second PR.